IntentToSecondScreen/src/androidTest/java/com/microsoft/device/display/samples/intentsecondscreen/OpenSecondActivityTest.kt

```diff
@@ -1,84 +0,0 @@
-/*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License.
- *
- */
-
-package com.microsoft.device.display.samples.intentsecondscreen
-
-import android.content.Intent
-import androidx.test.espresso.Espresso.onView
-import androidx.test.espresso.assertion.ViewAssertions.matches
-import androidx.test.espresso.intent.Intents.intended
-import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
-import androidx.test.espresso.intent.matcher.IntentMatchers.hasFlags
-import androidx.test.espresso.intent.rule.IntentsTestRule
-import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
-import androidx.test.espresso.matcher.ViewMatchers.withId
-import androidx.test.filters.LargeTest
-import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
-import com.microsoft.device.display.samples.intentsecondscreen.utils.forceClick
-import com.microsoft.device.display.samples.intentsecondscreen.utils.setOrientationLeft
-import com.microsoft.device.display.samples.intentsecondscreen.utils.switchFromSingleToDualScreen
-import com.microsoft.device.display.samples.intentsecondscreen.utils.unfreezeRotation
-import org.hamcrest.core.AllOf.allOf
-import org.junit.After
-import org.junit.Rule
-import org.junit.Test
-import org.junit.runner.RunWith
-
-@RunWith(AndroidJUnit4ClassRunner::class)
-@LargeTest
-class OpenSecondActivityTest {
```

Comment:
Why was this test class removed? I think the tests are still valid

Reply:
The class is still in the project, I think it was moved.